docs/guides/integration-javascript-client-side.md

```diff
@@ -214,7 +214,9 @@ In some cases, the user's personal data is not available on page load, and getti
 You can potentially avoid that cost by checking for an existing token that you can use or refresh. To do this, call
 [__euid.isLoginRequired](../sdks/sdk-ref-javascript#isloginrequired-boolean) which returns a Boolean value. If it returns `true`, this means that the EUID SDK cannot create a new advertising token with the existing resource and personal data is required to generate a brand new EUID token.
 
-The following code snippet demonstrates how you might integrate with the SDK for JavaScript for the two scenarios above&#8212;starting with no token as well as reusing/refreshing any existing EUID token if found. 
+It is possible that when you provide the user's personal data, [__euid.isLoginRequired](../sdks/sdk-ref-javascript#isloginrequired-boolean) still returns a `false` value. This happens if the user has opted out of EUID. The EUID SDK for JavaScript respects the user's optout and does not generate EUID tokens, even if you call any of the `setIdentity` method calls with the same user's personal data again. Optionally, you might want to avoid making such calls repeatedly.
```

Comment:
... does not generate EUID tokens...
update to 
... does not generate an EUID token...

Reply:
Made this edit in branch for update per Sunny's request.